dlp/snippets/jobs/jobs_test.go

```diff
@@ -64,19 +64,19 @@ func riskNumerical(projectID, dataProject, pubSubTopic, pubSubSub, datasetID, ta
 	ctx := context.Background()
 	client, err := dlp.NewClient(ctx)
 	if err != nil {
-		return fmt.Errorf("dlp.NewClient: %w", err)
+		return fmt.Errorf("dlp.NewClient: %v", err)
```

Comment:
Revert. `%w` is the correct formatting directive.

Reply:
fixed it.